lib/tasks/change_time_type_of_enrollments.rake

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+desc 'change time type of enrollments attributes'
+task change_time_type_of_enrollments: :environment do
+  puts '****************************************************************************************************'
+  puts '*                               start migration                                                    *'
+  puts '****************************************************************************************************'
```

Comment:
If you want, you can use Ruby's `String#ljust`, `String#rjust`, and `String#center`.
e.g.:
```ruby
puts ''.ljust 80, '*'
puts 'start migration'.center 80
puts ''.ljust 80, '*'
```
if you find it easier to write